ci/playbooks/deploy_edpm.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- hosts: controller
+  vars:
+    install_yamls_basedir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+  tasks:
+    - name: Install libguestfs-tools-c package
+      become: true
+      package:
+        name: libguestfs-tools-c
+        state: present
+
+    - name: Run make edpm_compute
+      community.general.make:
+        target: edpm_compute
+        chdir: "{{ install_yamls_basedir }}/devsetup"
+        params:
+          OUT: "{{ install_yamls_basedir }}/out"
+          EDPM_COMPUTE_SUFFIX: 0
+
+    - name: Sleep for 60 secs to get compute node up
+      ansible.builtin.shell: |
+        sleep 60;
+
+    - name: Run make edpm_compute_repos
+      community.general.make:
+        target: edpm_compute_repos
+        chdir: "{{ install_yamls_basedir }}/devsetup"
+        params:
+          EDPM_COMPUTE_SUFFIX: 0
+
+    - name: Run EDPM Deploy
+      community.general.make:
+        target: edpm_deploy
+        chdir: "{{ install_yamls_basedir }}"
+        params:
+          OUT: "{{ install_yamls_basedir }}/out"
+          DATAPLANE_CHRONY_NTP_SERVER:  pool.ntp.org
+
+    - name: Get info about dataplane node
+      ansible.builtin.shell: |
+        sleep 60;
+        oc get openstackdataplane
+        oc get openstackdataplanerole
+        oc get openstackdataplanenode
+        oc get pods | grep edpm
+      ignore_errors: true
+
+    - name: Get dataplane-deployment-run pod name
+      register: edpm_pod
+      ansible.builtin.shell:
+        cmd: oc get pods -o name | grep -m1  dataplane-deployment-run-openstack
```

Comment:
So other `dataplane-deployment` AEE pods will run but you're waiting for `dataplane-deployment-run-openstack`, which is the last one as defined in deployment.go [1]. That is not the last AEE to run however, as when that finishes two nova playbooks run in parallel [2]. When that ansible is finished running (with two edpm compute nodes) it looks like this [3].

Maybe it's more correct, if the intent is to be sure all ansible jobs have run, to wait for the job matching `nova-edpm-compute-0-deploy-nova` to be finished instead of the one matching `dataplane-deployment-run-openstack`.

I see however, that we have green jobs so we don't have to block on this. It could lead to more stable CI however to be sure all relevant AEE jobs have run before running the validation. I suspect the green jobs worked but it's probably because they won the race.

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/pkg/deployment/deployment.go#L329

[2] https://github.com/openstack-k8s-operators/nova-operator/tree/master/playbooks

[3]
```
[fultonj@osp-storage-01 devsetup]$ oc get pods | egrep "dataplane|edpm"
dataplane-deployment-configure-network-f7qpm-mj9dp                0/1     Completed                0                 9m24s
dataplane-deployment-configure-openstack-hnzwf-9prkt              0/1     Completed                0                 3m23s
dataplane-deployment-configure-os-x7g52-vw79w                     0/1     Completed                0                 5m44s
dataplane-deployment-install-openstack-pdr9v-d9bcv                0/1     Completed                0                 3m43s
dataplane-deployment-install-os-k5v9r-d94rb                       0/1     Completed                0                 6m35s
dataplane-deployment-run-openstack-kh5d8-hbxbr                    0/1     Completed                0                 2m57s
dataplane-deployment-run-os-jndpm-r5fxc                           0/1     Completed                0                 4m25s
dataplane-deployment-validate-network-kns2z-wdl2z                 0/1     Completed                0                 6m45s
nova-edpm-compute-0-deploy-libvirt-gh4hc                          0/1     Completed                0                 2m24s
nova-edpm-compute-0-deploy-nova-gxh7w                             0/1     Completed                0                 2m24s
nova-edpm-compute-1-deploy-libvirt-kxsf6                          0/1     Completed                0                 2m24s
nova-edpm-compute-1-deploy-nova-h5glm                             0/1     Completed                0                 2m24s
[fultonj@osp-storage-01 devsetup]$ 
```

Reply:
For example...

`for POD in $(oc get pods -o name | egrep "dataplane|edpm"); do echo $POD; oc logs $POD; done`

If one of the many playbooks fail, we'll want their logs.

Reply:
Thank you @fultonj for the nice suggestion, I have made the changes.